int8 status
int64 score_home
int64 score_away